src/screens/LocationDetailsScreen.tsx

```diff
@@ -1,19 +1,369 @@
+import { SuccessModal } from "@components/SuccessModal";
+import { renderLastUpdated } from "@utils/waitTime";
+
+export const LocationDetailsScreen = () => {
+  const { theme } = useContext(ThemeContext);
+  const [refreshing, setRefreshing] = useState(false);
+  const [showSubmittedSuccessModal, setSubmittedSuccessModal] = useState(false);
+  const [showConfirmSuccessModal, setConfirmSuccessModal] = useState(false);
+  const [newWaitTime, setNewWaitTime] = useState(0);
+
+  const chartData = [
+    { quarter: 7, earnings: 1 },
+    { quarter: 8, earnings: 2 },
+    { quarter: 9, earnings: 1 },
+    { quarter: 10, earnings: 4 },
+    { quarter: 11, earnings: 6 },
+    { quarter: 12, earnings: 15 },
+    { quarter: 13, earnings: 6 },
+    { quarter: 14, earnings: 4 },
+    { quarter: 15, earnings: 5 },
+    { quarter: 16, earnings: 5 },
+    { quarter: 17, earnings: 7 },
+    { quarter: 18, earnings: 3 },
+    { quarter: 19, earnings: 15 },
+    { quarter: 20, earnings: 3 },
+    { quarter: 21, earnings: 2 },
+    { quarter: 22, earnings: 5 },
+    { quarter: 23, earnings: 1 },
+  ];
+
+  const poiData = {
+    name: "Centro",
+    address: "Commons Building",
+    distance: 2.1,
+    hours: [
+      { "Monday - Friday": "7:00 AM - 11:00 PM" },
+      { "Saturday & Sunday": "10:00 AM - 11:00 PM" },
+    ],
+    lastUpdated: 4,
+    waitTime: 10,
+    currentHour: 14,
+  };
+
+  const submitNewWaitTime = () => {
+    // send new wait time to backend
+    // api call with newWaitTime
+    setSubmittedSuccessModal(true);
+    setTimeout(() => {
+      setSubmittedSuccessModal(false);
+    }, 2000);
+  };
+
+  const confirmNewWaitTime = () => {
+    // send wait time to backend
+    setConfirmSuccessModal(true);
+    setTimeout(() => {
+      setConfirmSuccessModal(false);
+    }, 2000);
+  };
+
+  const onRefresh = useCallback(() => {
+    setRefreshing(true);
+    setTimeout(() => {
+      setRefreshing(false);
+    }, 1000);
+  }, []);
+
+  const popularTimeChart = (
+    <VictoryChart height={175}>
+      <VictoryAxis
+        // tickValues specifies both the number of ticks and where
+        // they are placed on the axis
+        // TODO: Will need to change this to be dynamic based on the hours of operation
+        tickValues={[
+          7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23,
+        ]}
+        tickFormat={[
+          "7am",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "3pm",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "",
+          "11pm",
+        ]}
+        style={{
+          tickLabels: { fill: theme.styles.placeholderText.color },
+          axis: { stroke: "none" },
+        }}
+      />
+      <VictoryBar
+        style={{
+          data: {
+            fill: "#1677FF",
+            opacity: ({ datum }) =>
+              datum.quarter === poiData.currentHour ? 1 : 0.5,
```

Comment:
Is there an easy way to make this not depend on the api and instead pull the current hour from the device's time?